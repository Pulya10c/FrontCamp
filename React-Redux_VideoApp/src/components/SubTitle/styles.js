import styled from 'styled-components';
import { mediaMd, mediaSm } from 'utils/style-utils';

export const SubTitle = styled.h3`
  font-size: 20px;

  ${mediaMd`
    font-size: 18px;
  `};

  ${mediaSm`
    font-size: 14px;
  `};
`;
