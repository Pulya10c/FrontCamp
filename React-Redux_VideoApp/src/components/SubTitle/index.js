import { SubTitle } from './styles';

export default SubTitle;
