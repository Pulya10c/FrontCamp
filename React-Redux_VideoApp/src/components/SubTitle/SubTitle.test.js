import React from 'react';
import { shallow } from 'enzyme';
import SubTitle from './';

describe('Component SubTitle', () => {
  it('should render correctly', () => {
    expect(shallow(<SubTitle />)).toMatchSnapshot();
  });
});
