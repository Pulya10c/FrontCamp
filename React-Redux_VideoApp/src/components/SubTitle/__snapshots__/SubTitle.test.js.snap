// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component SubTitle should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  font-size: 20px;

  ",
          "
  @media (max-width: ",
          "768",
          "px) {
    ",
          "
    font-size: 18px;
  ",
          ";
  }
",
          ";

  ",
          "
  @media (max-width: ",
          "576",
          "px) {
    ",
          "
    font-size: 14px;
  ",
          ";
  }
",
          ";
",
        ],
      },
      "displayName": "styled.h3",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "h3",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
